MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+include py.typed
```

Comment:
I think this is the equivalent of having
```python
setup(
    ...
    package_data={'cylc.uiserver': ['py.typed']}
)
```
in `setup.py`

Reply:
Yep, works for the other files.